test/unittest/CMakeLists.txt

```diff
@@ -115,6 +140,9 @@ foreach(blas_test ${SYCL_UNITTEST_SRCS})
   if(is_adaptivecpp AND ${blas_test} IN_LIST ADAPTIVE_CPP_SKIP)
     continue()
   endif()
+  if(${blas_test} IN_LIST DEFAULT_SKIP)
+    continue()
+  endif()
```

Comment:
Having a single list of things to skip seems like better design to me. I'd prefer something like:
```
if(${blas_test} IN_LIST TESTS_TO_SKIP)
```
where `TESTS_TO_SKIP` is set outside the loop, instead of having `is_adaptivecpp` and a default-empty `DEFAULT_SKIP`.

If this is a real pain to retest, this isn't that important for now. It just feels like its heading in a messy direction.

Reply:
Testing using `DEFAULT` makes everything easier so I did it. I agree with you and avoiding messy situation is worthed. Addressed in 2b209ec9ba0164f745dc0d446cda11ecb2f01736